### PR TITLE
fix(ticket): dont show closed by meta for tickets in request status

### PR DESCRIPTION
**Pull request by wescopeland**
This pull request addresses a bug in the renderTicketCard function, which caused an incorrect display when the closedBy value of a ticket was not truthy. The issue resulted in an empty "Closed by" name and a close date showing"01 Jan 1970".

ref: https://discord.com/channels/310192285306454017/1098327873686880276/1098327873686880276

### CODE REVIEW

File: app_legacy/Helpers/render/ticket.php [hunk 1]
```diff
@@ -57,13 +57,17 @@ function renderTicketCard(int|Ticket $ticket): string
...
+        : '';
+
     return "<div class='tooltip-body flex items-start' style='max-width: 400px'>" .
         "<img style='margin-right:5px' src='" . media_asset('/Badge/' . $ticket->badgeName . '.png') . "' width='64' height='64' />" .
         "<div class='ticket-tooltip-info $ticketStateClass'>" .
         "<div><b>" . $ticket->achievementTitle . "</b> <i>(" . $ticket->gameTitle . ")</i></div>" .
         "<div>Reported by $ticket->createdBy</div>" .
         "<div>Issue: " . TicketType::toString($ticket->ticketType) . "</div>" .
-        "<div class='tooltip-closer'>Closed by $ticket->closedBy, " . getNiceDate(strtotime($ticket->closedOn)) . "</div>" .
+        $closedByInfo .
```
Comment (Jamiras) [inline app_legacy/Helpers/render/ticket.php:70]: The visibility of these fields is managed by css:

https://github.com/RetroAchievements/RAWeb/blob/e7c304be4371c4989a653e2216c8f7a906f74945/resources/css/ticket-manager.css#L25-L28

You just need to add `TicketState::request` to the `match` on line 55 (and line 25 for the embed border)

Comment (wescopeland) [reply app_legacy/Helpers/render/ticket.php]: Thank you, I've made this change in https://github.com/RetroAchievements/RAWeb/pull/1497/commits/eb2ca36830da31f13a4a9c218789a23df7e7451a. Not sure how I missed that.

Comment (Jamiras) [reply app_legacy/Helpers/render/ticket.php]: Please also make the change on line 25

Comment (wescopeland) [reply app_legacy/Helpers/render/ticket.php]: Addressed in https://github.com/RetroAchievements/RAWeb/pull/1497/commits/74b06ca93bf4e0143e990c75428bcaf9904b0bf3

**Review by Jamiras — APPROVED** [review]

**Pull request merged**